pkgs/applications/misc/alacritty/default.nix

```diff
@@ -47,6 +47,11 @@ buildRustPackage rec {
     pkgconfig
```

Comment:
We can probably remove xclip from buildInputs. I don't think it's used at build time.